include/color_spinor_field_order.h

```diff
@@ -562,6 +585,36 @@ namespace quda {
 #endif
       }
 
+      /**
+       * Read-only complex-member accessor function.  The last
+       * parameter n is only used for indexed into the packed
+       * null-space vectors.
+       * @param x 1-d checkerboard site index
+       * @param s spin index
+       * @param c color index
+       * @param v vector number
+       */
+      template <int nSpinBlock>
+      __device__ __host__ inline void load(complex<Float> out[nSpinBlock * nColor * nVec], int parity, int x_cb,
+                                           int chi) const
```

Comment:
It would be great if the documentation can be updated here.

Reply:
Good catch, will do.